deploy/rbd/kubernetes/csi-ceph-conf-override.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: csi-ceph-conf-override
+  namespace: rook-ceph  # namespace:operator
```

Comment:
Remove this one as we dont want to refer or assume that ceph-csi is deployed by Rook